domain/domain.go

```diff
@@ -711,78 +712,86 @@ func (do *Domain) refreshMDLCheckTableInfo() {
 }
 
 func (do *Domain) mdlCheckLoop() {
-	ticker := time.Tick(time.Millisecond * 50)
+	var ticker <-chan time.Time
+	if intest.InTest {
```

Comment:
If we provide a notify after MDL updated is change the tick duration in test required? @hawkingrei

Reply:
I think it is possible to implement it.